drivers/at86rf215/at86rf215_netdev.c

```diff
@@ -199,6 +199,15 @@ static int _recv(netdev_t *netdev, void *buf, size_t len, void *info)
     if (info != NULL) {
         netdev_ieee802154_rx_info_t *radio_info = info;
         radio_info->rssi = (int8_t) at86rf215_reg_read(dev, dev->RF->RG_EDV);
+
+        /* read timestamp counter */
+        uint32_t rx_timestamp;
+        at86rf215_reg_read_bytes(dev, dev->BBC->RG_CNT0, &rx_timestamp,
+                                sizeof(rx_timestamp));
+
+        /* convert counter value to ns */
+        radio_info->timestamp = rx_timestamp * 10^3 / 32;
```

Comment:
`^` is [bitwise XOR](https://en.wikipedia.org/wiki/Bitwise_operations_in_C#Bitwise_XOR_^), just spell it out.

```suggestion
        radio_info->timestamp = (rx_timestamp * 1000) / 32;
```

Reply:
The timestamp could be read again when evaluating a time synchronization message to compensate for the delay. I guess it would be also good to read the timestamp when sending, though.